public/js/main.js

```diff
@@ -59,26 +59,26 @@ async function changeStatusFalse() {
   }
 }
 
-// Cafe: Delete shift
-Array.from(deleteShiftBtn).forEach(btn => {
-  btn.addEventListener('click', deleteShift)
+// Cafe: remove shift from being displayed on cafe dashboard
+Array.from(removeShiftBtn).forEach(btn => {

```

Comment:
Looks like you don't need to cast this to an array to iterate it https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach